app/index.js

```diff
@@ -81,6 +82,11 @@ function onListening() {
   const addr = server.address();
   const bind = typeof addr === 'string' ? `pipe ${addr}` : `port ${addr.port}`;
   log.info(`razeedash-api listening on ${bind}`);
+
+  // Only if AUTH_MODEL is defined, then enable graphql query API.
+  if (process.env.AUTH_MODEL) {
```

Comment:
Can we use a better name to enable Apollo.  This will get confusing in the future.  Something like `ENABLE_GRAPHQL` or something.  Then if that is set but `AUTH_MODEL` is not, then throw an error an exit.

Reply:
Add ENABLE_GRAPHQL environment variable , default is false. see [here](https://github.com/razee-io/Razeedash-api/blob/graphql/app/index.js#L87) and [here](https://github.com/razee-io/Razeedash-api/blob/graphql/app/routes/v2/clusters.js#L243)